Userland/Applications/PartitionEditor/PartitionModel.h

```diff
@@ -22,7 +22,7 @@ class PartitionModel final : public GUI::Model {
         __Count,
     };
 
-    static NonnullRefPtr<PartitionModel> create() { return adopt_ref(*new PartitionModel()); }
+    static NonnullRefPtr<PartitionModel> create();
```

Comment:
This seems completely unrelated to the commit

Reply:
Definitely seems that way at a first glance, but see my previous comment here. If this needs its own commit or something let me know!

>All were very simple changes to make. Done! Had to >move `PartitionModel::create()` out of line to get it to >stop complaining about the forward declaration of >`DeprecatedFile`
>
>~~The only issue is the `DeprecatedFile` header needs >to be declared in `PartitionModel.h` otherwise, it fails >to compile. (Complains about something to do with >ForwardDeclaring and unref-ing `DeprecatedFile`)~~

Reply:
ah, fair enough :)